magicalgirlbossfight/src/magicalgirlbossfight/players.py

```diff
@@ -0,0 +1,36 @@
+import ppb
+from ppb import keycodes
+from ppb.events import KeyPressed, KeyReleased, Update
+
+
+class Player(ppb.Sprite):
+
+    def __init__(self, image=ppb.Image('magicalgirlbossfight/resources/magicalgirlbossfight.png'),
```

Comment:
So mentioned this previously, but this section is breaking a lot of ppb expectations. In general, for these kinds of defaults you want to set them as class attributes.

```python
class Player(ppb.Sprite):
    image = ppb.Image('magicalgirlbossfight/resources/magicalgirlbossfight.png')
    position = ppb.Vector(0, 0)
    move_vector = ppb.Vector(0, 0)
```

With this, we don't need the `__init__` function at all, and it gives us way more flexibility during experimentation by using the ppb `__init__` that lets us add data at call time.